.travis.yml

```diff
@@ -23,6 +23,7 @@ env:
     - SFDX_AUTOUPDATE_DISABLE=true
     - SFDX_USE_GENERIC_UNIX_KEYCHAIN=true
     - SFDX_DOMAIN_RETRY=300
+    - CODE_TESTS_WORKSPACE=$TRAVIS_BUILD_DIR/packages/system-tests/sfdx-simple
```

Comment:
I don't think this is necessary. What happens if you remove it?

If this were necessary, then my PR from yesterday that had the integration tests would have failed.